deploy/Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM quay.io/openshift/origin-operator-registry:latest
 
-COPY deploy/olm-catalog /registry
+WORKDIR /registry
+
+COPY deploy/olm-catalog .
+
+USER root
```

Comment:
Could you please elaborate on why is that necessary?

Reply:
Sure, tl;dr - https://github.com/operator-framework/operator-registry/pull/430 introduced a change that breaks our image.  Previously /registry was created in the operator-registry image, and had g+rwx permissions, so we could create and use /registry/bundles.db.  This is not the case anymore, so when running initializer, the bundles.db can't be written to /registry.  In addition to that, there is not /bin/sh, so RUN should be used with exec and not with `sh -c` - they are working on a fix in https://github.com/operator-framework/operator-registry/pull/435 but with this change, we won't depend on /bin/sh